private int sum(int a, int b) throws Exception(){
if((a+b)>Integer.Max){
throw new Exception();
}
return a+b;
}